demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -1302,21 +1352,44 @@ def test_modeling_rule(
             interactive,
             ctx,
         )
-        if not success:
+        if success:
+            logger.info(
+                f"[green]Test Modeling rule {get_relative_path_to_content(modeling_rule_directory)} passed[/green]",
+                extra={"markup": True},
+            )
+        else:
             errors = True
             logger.error(
-                f"[red]Error testing modeling rule {get_relative_path_to_content(modeling_rule_directory)}[/red]",
+                f"[red]Test Modeling rule {get_relative_path_to_content(modeling_rule_directory)} failed[/red]",
                 extra={"markup": True},
             )
         if modeling_rule_test_suite:
             modeling_rule_test_suite.add_property("start_time", start_time)
             xml.add_testsuite(modeling_rule_test_suite)
 
     if output_junit_file:
+        logger.info(
+            f"[cyan]Writing JUnit XML to {output_junit_file}[/cyan]",
```

Comment:
JUnit is Java, but we won't change it now. It is what it is...

Reply:
https://www.ibm.com/docs/en/developer-for-zos/14.1?topic=formats-junit-xml-format
junit is a file type format. not just for java.